administration/src/util/getDeepLinkFromQrCode.ts

```diff
@@ -1,18 +1,18 @@
+import { BuildConfigType } from 'build-configs'
 import { ACTIVATION_FRAGMENT, ACTIVATION_PATH, HTTPS_SCHEME } from 'build-configs/constants'
 
 import { PdfQrCode } from '../cards/pdf/PdfQrCodeElement'
 import { QrCode } from '../generated/card_pb'
 import { uint8ArrayToBase64 } from './base64'
-import { getBuildConfig } from './getBuildConfig'
 import { isProductionEnvironment } from './helper'
 
-const getDeepLinkFromQrCode = (qrCode: PdfQrCode): string => {
+const getDeepLinkFromQrCode = (qrCode: PdfQrCode, buildConfig: BuildConfigType, isProduction: boolean): string => {
```

Comment:
Instead of conditional logging, we could also remove log statements altogether when building for production:

https://medium.com/@shaangontia/remove-console-statements-webpack-5-38455e4b471a

Reply:
Hm we also have some logs for testing on staging environment to get the deep links so not sure if entirely removing them is the solution, but we could distinguish between staging and production